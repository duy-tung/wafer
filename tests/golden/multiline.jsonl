{"id":"550e8400-e29b-41d4-a716-446655440000","source_file":"multiline.txt","chunk_index":0,"text":"This is a multi-line test file for golden file testing. It contains multiple paragraphs and line breaks to test how the wafer CLI tool handles different text structures and formatting. The third paragraph includes some special characters: !@#$%^&*()_+-={}[]|;':\",./<>? This ensures comprehensive testing of the text processing pipeline.","embedding":[0.1,0.2,0.3,0.4,0.5],"word_count":50,"created_at":"2024-01-15T10:30:45Z"}
