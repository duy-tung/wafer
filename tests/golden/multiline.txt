This is a multi-line test file for golden file testing.

It contains multiple paragraphs and line breaks to test how the wafer CLI tool handles different text structures and formatting.

The third paragraph includes some special characters: !@#$%^&*()_+-={}[]|;':",./<>?

This ensures comprehensive testing of the text processing pipeline.
